filebeat/tests/system/config/filebeat.yml.j2

```diff
@@ -132,11 +133,16 @@ processors:
 # Multiple outputs may be used.
 
 #------------------------------- File output ----------------------------------
+{%- if logstash %}
```

Comment:
as template variable `logstash.host` is used to configure the hostname, I'd keep it like this.